### PR TITLE
A0-1352: Add clean script

### PR DESCRIPTION
**Pull request by fbielejec**
# Description

Script that terminates and removes all game contracts from chain

### CODE REVIEW

File: bin/cliain/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+**/target
+target
+docker/data
+!target/release/cliain
```
Comment (timorl) [inline bin/cliain/.dockerignore:4]: NIT: Maybe end the file with a newline, it's polite.

**Review by timorl — APPROVED** [review]
LGTM, although this is not a part of code I'm particularly familiar with.

File: contracts/scripts/clean.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# --- FUNCTIONS
+
+function terminate_contract {
+  local contract_name=$1
+  local contract_dir=$2
```
Comment (Marcin-Radecki) [inline contracts/scripts/clean.sh:9]: Quotes around variables that contains filesystem names please :)

**Review by Marcin-Radecki — APPROVED** [review]
Nice

File: contracts/scripts/deploy.sh [hunk 3]
```diff
@@ -194,8 +192,10 @@ cargo contract build --release
 
 cd "$CONTRACTS_PATH"/access_control
 
-CONTRACT=$(cargo contract instantiate --url "$NODE" --constructor new --suri "$AUTHORITY_SEED")
-ACCESS_CONTROL=$(echo "$CONTRACT" | grep Contract | tail -1 | cut -c 15-)
+ACCESS_CONTROL_CODE_HASH=$(cargo contract upload --url "$NODE" --suri "$AUTHORITY_SEED")
+ACCESS_CONTROL_CODE_HASH=$(echo "$ACCESS_CONTROL_CODE_HASH" | grep hash | tail -1 | cut -c 15-)
+ACCESS_CONTROL=$(cargo contract instantiate --url "$NODE" --constructor new --suri "$AUTHORITY_SEED")
+ACCESS_CONTROL=$(echo "$ACCESS_CONTROL" | grep Contract | tail -1 | cut -c 15-)
```
Comment (Marcin-Radecki) [inline contracts/scripts/deploy.sh:198]: Those `cut`s after `N` columns tends to break frequently as we depend on format of some external output command like `cargo contract` in this example, but I guess we don't have a better way for now.

File: contracts/scripts/clean.sh (hunk 2 above)
Comment (fbielejec) [reply contracts/scripts/clean.sh:9]: done

File: bin/cliain/.dockerignore (hunk 1 above)
Comment (fbielejec) [reply bin/cliain/.dockerignore:4]: done

**Pull request merged**